infra/base-images/base-builder/precompile_centipede

```diff
@@ -20,7 +20,8 @@ echo -n "Precompiling centipede"
 # Build Centipede with bazel.
 cd "$SRC/centipede/"
 ./install_dependencies_debian.sh
-bazel build -c opt :all
+bazel --bazelrc="$SRC/centipede/.bazelrc" --bazelrc=/tmp/centipede.bazelrc \
```

Comment:
Do we still need the explicit `--bazelrc="$SRC/centipede/.bazelrc"` ? or is that automatically added?

Reply:
Yes, it is automatically added. Thanks!